features/polyExplorer/src/screens/entityDetails/entityDetails.jsx

```diff
@@ -19,17 +19,17 @@ const EntityDetails = () => {
     const [swiper, setSwiper] = useState(null);
 
     const cityImageMap = {
-        München: "munich",
-        "Mountain View": "mountainview",
-        Wiesbaden: "wiesbaden",
-        Berlin: "berlin",
-        Dubai: "dubai",
-        Luxembourg: "luxembourg",
-        "WILMINGTON, New Castle": "wilmington",
-        "WILMINGTON, DELAWARE": "wilmington",
-        WILMINGTON: "wilmington",
-        Cupertino: "cupertino",
-        DUBLIN: "dublin",
+        münchen: "munich",
+        "mountain view": "mountainview",
+        wiesbaden: "wiesbaden",
+        berlin: "berlin",
+        dubai: "dubai",
+        luxembourg: "luxembourg",
+        "wilmington, new castle": "wilmington",
```

Comment:
Is New Castle a thing?

Reply:
Well, apparently it's the county where Wilmington is...

Reply:
Yeah some have this whole string instead of Wilmington as location